walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -30,4 +30,22 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
+
+  @Test
+  public void testGreetingTrimsWhitespace() {
+    Greeter greeter = new Greeter();
```

Comment:
while this exercise is pretty simple, you may want to give fixtures a try: https://junit.org/junit4/cookbook.html